source/Weapon.cpp

```diff
@@ -151,7 +151,9 @@ void Weapon::LoadWeapon(const DataNode &node)
 				turn = value;
 			else if(key == "inaccuracy")
 				inaccuracy = value;
-			else if(key == "turret turn")
+			else if(key == "inaccuracy smoothness")
+				inaccuracySmoothness = value;
+		   else if(key == "turret turn")
```

Comment:
Indents.

Reply:
Fixed in 32a94702fd9a64473f9f49d4c213a379656d3497.